test/spec/lfe2e/specs/basic_w_custom_adserver_translation.spec.js

```diff
@@ -0,0 +1,64 @@
+const includes = require('core-js/library/fn/array/includes');
+const expect = require('chai').expect;
+
+const ACCEPTED_DURAS = ['15s', '30s'];
+const ACCEPTED_CATS = ['Food', 'Retail Stores/Chains', 'Pet Food/Supplies', 'Travel/Hotels/Airlines', 'Automotive', 'Health Care Services'];
+const ACCEPTED_CPMS = ['15.00', '14.00', '13.00', '10.00'];
+const customKeyRegex = /\d{2}\.\d{2}_\d{1,3}_\d{2}s/;
+const uuidRegex = /(\d|\w){8}-((\d|\w){4}-){3}(\d|\w){12}/;
+
+describe('longform ads using custom adserver translation file', function() {
+  this.retries(3);
+  it('process the bids successfully', function() {
+    browser
+      .url('http://test.localhost:9999/integrationExamples/longform/basic_w_custom_adserver_translation.html?pbjs_debug=true')
```

Comment:
What happens if test.localhost is not defined in my hosts file ?

Reply:
I will test this and determine if it's an issue.